main/Dockerfile

```diff
@@ -6,6 +6,7 @@ ARG AEM_ADDR
 ARG DOMAIN
 ARG S3_ADDR
 ARG STORYLINES_ADDR
+ARG REDIS_PORT_6379_TCP_ADDR_A
```

Comment:
I think this also needs to be added to https://github.com/CruGlobal/ecs_config/blob/master/ecs/base-image-openresty/env.conf

Reply:
The value isn't used by nginx at runtime, so it doesn't need to be there, but it did need to be added to build.sh, which I've now down.